File: Day 06 - Let's Review/solution.js
function processData(input) {
  //Enter your code here
  var mensaje = input.split("\n");
  for (var i = 1; i < mensaje.length; i++) {
    var aux1 = "";
    var aux2 = "";
    for (var j = 0; j < mensaje[i].length; j++) {
      if (j % 2 == 0) {
        aux1 = aux1 + mensaje[i][j];
      } else {
        aux2 = aux2 + mensaje[i][j];
      }
    }
    console.log(aux1 + " " + aux2);
  }
}

File: Day 06 - Let's Review/solution.py
# Enter your code here. Read input from STDIN. Print output to STDOUT
number_words = int(input())
for i in range(number_words):
    even_letters = ""
    odd_letters = ""
    word = input()

    for j in range(len(word)):
        if (j % 2 == 0):
            even_letters = even_letters + word[j]
        else:
            odd_letters = odd_letters + word[j]

    print(even_letters, odd_letters)

# Option 2
# number_words = int(input())
# for i in range(number_words):
#    word = input()
#    print(word[::2], word[1::2])
